packages/lodestar/src/sync/utils/sync.ts

```diff
@@ -211,7 +211,10 @@ export function createStatus(chain: IBeaconChain): Status {
 
 export async function syncPeersStatus(reps: IReputationStore, network: INetwork, status: Status): Promise<void> {
   await Promise.all(network.getPeers().map(async (peerId) => {
-    reps.get(peerId.toB58String()).latestStatus = await network.reqResp.status(peerId, status);
+    try {
+      reps.get(peerId.toB58String()).latestStatus = await network.reqResp.status(peerId, status);
+      // eslint-disable-next-line no-empty
+    } catch {}
```

Comment:
Is this error ignorable? Can it be logged to verbose or lower?

Reply:
It's gonna be logged in network module. We cannot handle it here. It will be updated in next round